src/ansys_sphinx_theme/theme/ansys_sphinx_theme/_templates/autosummary/class.rst

```diff
@@ -26,9 +26,7 @@
    .. autosummary::
       :toctree:
    {% for item in attributes %}
-      {% if item.0 != item.upper().0 %}
       {{ name }}.{{ item }}
-      {% endif %}
```

Comment:
@Revathyvenugopal162 Did you check that it fixed the enum issue in some local documentation without any (obvious) side effects?
I think it should be all good.

Reply:
That's a good point, @MaxJPRey.

Reply:
I checked, and earlier the autosummary will not render the attributes in uppercase, and now it will support both upper and lower case( especially with enum-class members.)
## Rendered new docs.
![enum_new](https://user-images.githubusercontent.com/104772255/216913912-cb7a48bc-4b53-48a0-b653-ff3c0388cd9f.PNG)

Reply:
Also, [PyDF-composite](https://composites.dpf.docs.pyansys.com/release/0.1/api/_autosummary/ansys.dpf.composites.constants.Sym3x3TensorComponent.html) is using the same behavior now.

Reply:
Perfect. Thanks @Revathyvenugopal162 .